Dockerfile-ubuntu

```diff
@@ -0,0 +1,74 @@
+# Copyright © 2016 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Cask is a trademark of Cask Data, Inc. All rights reserved.
+
+FROM ubuntu:12.04
+MAINTAINER Cask Data <ops@cask.co>
+
+# Copy files
+# Setup ENV
+# Install Chef
+# Setup chef dir
+# Install/run omnibus, java, and maven cookbooks
+
+COPY . /var/tmp/coopr-build
+
+ENV PATH /usr/local/bin:${PATH}
+ENV COOPR_BUILD_PROJECTS coopr-cli coopr-provisioner coopr-server coopr-ui
+
+RUN useradd -m coopr && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    curl \
+    git \
+    python2.7 && \
```

Comment:
We do. Ubuntu ships 2.7 or 3.0... and this is a dependency of a dependency. No Coopr code is Python.